acceptance/tests/security/cert_whitelist.rb

```diff
@@ -24,7 +24,7 @@
                  "--cacert #{ssldir}/certs/ca.pem " +
                  "--cert #{ssldir}/certs/#{dbname}.pem " +
                  "--key #{ssldir}/private_keys/#{dbname}.pem "+
-                 "https://#{dbname}:8081/v3/metrics/mbeans " +
+                 "https://#{dbname}:8081/metrics/v1/mbeans " +
```

Comment:
This is good, but there are two other places in acceptance/helper.rb that will need a similar change also. Grep for 'mbean' (not mbeans) and you'll see what I mean.